tasks/publish.js

```diff
@@ -22,6 +22,8 @@ const publish = async (directory) => {
     if (process.env.DRY_RUN) {
       log.warn('Dry run! To publish the release, run the command again without DRY_RUN environment variable')
     }
+    // copy packages/core/types/index.d.ts to packages/asciidoctor/types/index.d.ts
+    fs.copyFileSync(`${projectRootDirectory}/packages/core/types/index.d.ts`, `${projectRootDirectory}/packages/asciidoctor/types/index.d.ts`)
```

Comment:
I've decided to copy the `index.d.tx` just before publishing the `asciidoctor` package. I don't think we need to version this file a second time.
I'm not sure if a file link works so I prefer to just copy the file.

It's mandatory otherwise the TypeScript Declaration File won't be found in the example below:

```js
import Processor from 'asciidoctor'
```

Reply:
`projectRootDirectory` is undefined here! 🐛